docs/reference/query-languages/index.md

```diff
@@ -5,20 +5,12 @@ applies_to:
 ---
 # Query languages
 
-:::{note}
-This section provides detailed **reference information** for query languages.
-
-Refer to [Query Languages](docs-content://explore-analyze/query-filter/languages.md) in the **Explore and analyze** section for overview, getting started and conceptual information.
-:::
-
-This section contains reference information for Elastic query languages.
+This section contains reference information for Elastic query languages, including:
 
 * [Query DSL](querydsl.md)
 * [ES|QL](esql.md)
 * [SQL](sql.md)
 * [EQL](eql.md)
 * [KQL](kql.md)
 
-:::{tip}
-Refer to [query languages](docs-content://explore-analyze/query-filter/languages.md) in the **Explore and analyze** section for overview and conceptual information about each language.
-:::
\ No newline at end of file
+For more information about each language, check out the [query languages](docs-content://explore-analyze/query-filter/languages.md) documentation.
```

Comment:
```suggestion
For more information about each language, refer to the [Explore and analyze](docs-content://explore-analyze/query-filter/languages.md) section.
```

Reply:
nit: I find this a little more helpful in at least understanding that the the docs are distributed into different sections, i.e. **Reference** and **Explore and analyze**. 

"the query languages documentation" sounds a little generic to me.